tools/syntax_updater/from_0_0_6.rs

```diff
@@ -0,0 +1,51 @@
+/* LICENSE BEGIN
+    This file is part of the SixtyFPS Project -- https://sixtyfps.io
+    Copyright (c) 2020 Olivier Goffart <olivier.goffart@sixtyfps.io>
+    Copyright (c) 2020 Simon Hausmann <simon.hausmann@sixtyfps.io>
+
+    SPDX-License-Identifier: GPL-3.0-only
+    This file is also available under commercial licensing terms.
+    Please contact info@sixtyfps.io for more information.
+LICENSE END */
+
+use std::io::Write;
+
+use sixtyfps_compilerlib::object_tree;
+use sixtyfps_compilerlib::parser::{syntax_nodes, SyntaxNode};
+
+/// Remove colspan, rowpan, col and row for items not in a GridLayout
+pub(crate) fn fold_node(
+    node: &SyntaxNode,
+    file: &mut impl Write,
+    state: &mut crate::State,
+) -> std::io::Result<bool> {
+    if let Some(binding) = syntax_nodes::Binding::new(node.clone()) {
+        if matches!(
+            state.property_name.as_ref().map(String::as_str).unwrap_or_default(),
+            "colspan" | "rowspan" | "row" | "col"
+        ) {
+            if let Some(elem) =
+                find_parent_element(binding.into()).and_then(|x| find_parent_element(x.into()))
+            {
+                let elem_name = elem
+                    .QualifiedName()
+                    .map(|qn| object_tree::QualifiedTypeName::from_node(qn).to_string())
+                    .unwrap_or_default();
+                if !matches!(elem_name.as_str(), "GridLayout" | "Row") {
+                    write!(file, "/* {} // REMOVED BY THE SYNTAX UPDATOR */", node.to_string())?;
```

Comment:
Nitpick: updator -> updater :)

Reply:
Note: I was thinking perhaps we should print a warning from there.
Another thing that could be done is to add something like  `@warning("Syntax updater removed: colspan: 2")`